docs/all.md

```diff
@@ -495,6 +495,18 @@ cf run-task CLOUD-GOV-APP --command "flask command update-templates" --name YOUR
 
 [Here's more documentation](https://docs.cloudfoundry.org/devguide/using-tasks.html) about Cloud Foundry tasks.
 
+# Commands for test loading the local dev database
+
+All commands use the `-g` or `--generate` to determine how many instances to load to the db. The `-g` or `--generate` option is required and will always defult to 1. An example: `flask command add-test-uses-to-db -g 6` will generate 6 random users and insert them into the db.
+
+## Test commands list
+- `add-test-organizations-to-db`
+- `add-test-services-to-db`
+- `add-test-jobs-to-db`
+- `add-test-notifications-to-db`
+- `add-test-users-to-db` (extra options include `-s` or `--state` and `-d` or `--admin`)
+
+
```

Comment:
Awesome, thank you for adding this!

There's a table of contents section with links at the top of this file, it'd be good to make sure this section is included there so it's easy to find the new section. 🙂  You just need to add a link that has a bookmark to the heading; you'll see how the others are done and refer to the `div` `id`s.

Also, our `README` has a copy of the documentation table of contents that links to the `docs/all.md` file toward the bottom, it'd be good to make sure it's listed in there, too!  Note that the link in the `README` is slightly different because it needs to include the path to the file in the repo in the repo in addition to the heading shortcut itself.  Again, you'll see how the other links are constructed.

Let me know if anything is unclear about adding the table of contents links in this doc and the `README`!